ui/imports/shared/views/PasswordView.qml

```diff
@@ -132,54 +133,44 @@ Column {
         function isTooShort() { return newPswInput.text.length < root.minPswLen }
     }
 
-    spacing: 3 * Style.current.padding / 2
+    spacing: Style.current.bigPadding
     z: root.zFront
-    width: 416
 
     // View visual content:
     StatusBaseText {
         id: title
-        anchors.horizontalCenter: parent.horizontalCenter
+        Layout.alignment: Qt.AlignHCenter
         visible: root.titleVisible
         text: root.title
         font.pixelSize: 22
         font.bold: true
         color: Theme.palette.directColor1
     }
 
-    Column {
-        StatusBaseText {
-            id: introTxt
-            anchors.horizontalCenter: parent.horizontalCenter
-            text: root.introText
-            font.pixelSize: 12
-            color: Theme.palette.baseColor1
-        }
-
-        StatusBaseText {
-            id: recoverTxt
-            anchors.horizontalCenter: parent.horizontalCenter
-            text: root.recoverText
-            font.pixelSize: 12
-            color: Theme.palette.dangerColor1
-        }
+    StatusBaseText {
+        id: introTxtField
+        Layout.fillWidth: true
+        text: "%1 <font color=\"%2\">%3</font>".arg(root.introText).arg(Theme.palette.dangerColor1).arg(root.recoverText)
+        font.pixelSize: root.highSizeIntro ? 15 : 12
+        color: Theme.palette.baseColor1
+        wrapMode: Text.WordWrap
+        horizontalAlignment: TextEdit.AlignHCenter
     }
 
     // TODO replace with StatusInput as soon as it supports password
```

Comment:
Thanks for the fixes! <3 Now you can actually drop the TODO :)

Reply:
Oh.. leftover! Done!